hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/consumer/ConsumerMessageSenderTest.java

```diff
@@ -274,6 +276,38 @@ public void shouldBackoffRetriesWhenEndpointFails() throws InterruptedException
         verifyErrorHandlerHandleFailed(message, subscriptionWithBackoff, 1 + executionTime / senderBackoffTime);
     }
 
+    @Test
+    public void shouldBackoffRetriesOnRetryAfter() throws InterruptedException {
+        // given
+        int retrySeconds = 1;
```

Comment:
I understand it is not ideal for testing time, but I'd rather not invest in refactors driven by tests, while the core implementation accepts only seconds per [specification](https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html):
> The value of this field can be either an HTTP-date or an integer number of seconds (in decimal)